Scemantic Focus

```diff
@@ -0,0 +1,110 @@
+---
+id: 
+name: Scemantic Focus 
+rule_type: atomic
+
+description: | 
+ This rule checks that every element part of the sequential focus navigation has a semantic role.
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
```

Comment:
Should add CSS Styling, the sequential focus navigation is impacted by display:none and the like.

Reply:
This is not valid YAML. This needs to be indented.